mypy_protobuf/main.py

```diff
@@ -269,8 +269,24 @@ def _write_comments(self, scl: SourceCodeLocation) -> bool:
         if sci_loc.trailing_comments is not None:
             lines.extend(self._break_text(sci_loc.trailing_comments))
 
+        lines = [
+            # Replace backslashes with double-backslashes
+            # to remove anything that would become an escape sequence in a Python string.
+            # Escape triple-quotes that would otherwise end the docstring early.
+            line
+                .replace("\\", "\\\\")
```

Comment:
I believe we should use raw strings https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals - rather than trying to do this by hand.